### DISCUSSION TITLE
Importing maps from D drive

### DISCUSSION THREAD

**Discussion by hrfarmer** [ppy/osu#17004] [Q&A] [answered]
So on stable I set my songs folder to my D drive because I just don't have space for it on my C drive. If I run the beatmap import option, is lazer going to put my beatmaps on my D drive or will it put them on my C drive, because if it does my C drive it will fill up

**Comment by hlysine** [comment, marked as answer]
It will import to where your lazer `data` folder is, which is on C drive by default. Change the folder location in lazer before importing.

**Reply by peppy** [reply]
To elaborate, you can do this from the settings panel:

![osu! 2022-02-28 at 06 11 15](https://user-images.githubusercontent.com/191335/155933214-22f0f187-7624-467e-b9cd-6acff8f2972f.png)
